shuup/admin/modules/shops/__init__.py

```diff
@@ -85,7 +85,7 @@ def get_search_results(self, request, query):
 
         minimum_query_length = 3
         if len(query) >= minimum_query_length:
-            shops = Shop.objects.get_for_user(request.user).filter(translations__name__icontains=query)
+            shops = Shop.objects.get_for_user(request.user).filter(translations__name__icontains=query, status=1)
```

Comment:
`status=1`? Perhaps you have some status constant|enum|definition|whatever somewhere that could be used instead of magical number `1`?